### PR TITLE
Flesh out tagging

### PR DESCRIPTION
**Pull request by QuinnyPig**

### CODE REVIEW

**Comment by ThanosBaskous** [comment]
Great addition. Thanks @QuinnyPig! Bouncing through @jlevy or @max-zanko for a second review.

File: README.md [hunk 1]
```diff
@@ -494,6 +494,8 @@ So if you’re not going to manage your AWS configurations manually, what should
 	-	To label lifecycles, such as temporary resources or one that should be deprovisioned in the future
 	-	To distinguish production-critical infrastructure (e.g. serving systems vs backend pipelines)
 	-	To distinguish resources with special security or compliance requirements
+-       🔸For many years, there was a notorious 10 tag limit per resource, which could not be raised and caused many companies significant pain. As of 2016, this was [raised](https://aws.amazon.com/blogs/security/now-organize-your-aws-resources-by-using-up-to-50-tags-per-resource/) to 50 tags per resource.
+- In 2017, AWS introduced the ability to [enforce tagging](https://aws.amazon.com/blogs/aws/new-tag-ec2-instances-ebs-volumes-on-creation/) on instance and volume creation, deprecating portions of third party tools such as [Cloud Custodian](https://github.com/capitalone/cloud-custodian).
```
Comment (jlevy) [inline README.md]: This might be helpful enough it deserves a 🔹 ?

File: README.md [hunk 2]
```diff
@@ -494,6 +494,8 @@ So if you’re not going to manage your AWS configurations manually, what should
 	-	To label lifecycles, such as temporary resources or one that should be deprovisioned in the future
 	-	To distinguish production-critical infrastructure (e.g. serving systems vs backend pipelines)
 	-	To distinguish resources with special security or compliance requirements
+-       🔸For many years, there was a notorious 10 tag limit per resource, which could not be raised and caused many companies significant pain. As of 2016, this was [raised](https://aws.amazon.com/blogs/security/now-organize-your-aws-resources-by-using-up-to-50-tags-per-resource/) to 50 tags per resource.
```
Comment (jlevy) [inline README.md]: Technically this is not a 🔸 any more, maybe remove that emoji?

**Comment by QuinnyPig** [comment]
Made those changes.

**Comment by jlevy** [comment]
👍 Awesome!

**Pull request merged**